package-lock.json

```diff
@@ -1,18068 +1,7 @@
 {
```

Comment:
Why are there so many changes in this file? Looks suspicious.

Reply:
It's fine.  I think I might have mistakenly committed `lockfileVersion: 2` by mistake while playing around with yarn.  Leave it in the PR as we should use version 1 as that's what npm supports right now.